Code/autopkglib/FileFinder.py

```diff
@@ -16,13 +16,15 @@
 """See docstring for FileFinder class"""
 
 
+import os.path
 from glob import glob
-from autopkglib import Processor, ProcessorError
+
+from autopkglib import ProcessorError
```

Comment:
Why did it drop "Processor" from the import?

Reply:
Because it's unused anywhere in the recipe. FileFinder inherits from DmgMounter instead.

Reply:
OK. The GitHub code review tools make that difficult to investigate.